packages/core/src/dialects/abstract/query-generator.d.ts

```diff
@@ -158,6 +173,11 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
     options?: ArithmeticQueryOptions,
   ): string;
 
+  createTableQuery<M extends Model>(
+    tableName: TableName,
+    attributes: ModelAttributes<M, CreationAttributes<M>>,
```

Comment:
This type is incorrect, it does not accept ModelAttributes, it requires the final SQL

It's also the list of columns, not attributes

```
columns: { [columnName: string]: string },
```

Reply:
Good point, I was misguided by the usage of attributes in the JS code. Have updated the type and added a TODO so it can be fixed in the JS code upon migrating to TS